source/main.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import os
+os.environ['CUDA_LAUNCH_BLOCKING'] = "1"
+import numpy as np
+import pandas as pd
+pd.set_option('mode.chained_assignment', None) # remove pandas copy to slice warnings
+from argparse import ArgumentParser
+
+import torch
+from pytorch_transformers import AdamW
+from tensorboardX import SummaryWriter
+# High(er) level api to pytorch
+from ignite.engine import Engine, Events
+from ignite.metrics import Accuracy, Loss, RunningAverage, Precision, Recall
+from ignite.handlers import ModelCheckpoint, EarlyStopping
+from ignite.contrib.handlers import ProgressBar
+# Custom modules
+from models.bert import BertForWSD
+from models.xlnet import XLNetForWSD
+
+from dataloaders.dataloader_utils import gen_dataloader
+from ignite_extras.helper_classes import Ignite_Engines
+
+
+def score_function(engine):
```

Comment:
all of these functions should be defined elsewhere and imported, e.g. as I did in the referenced main.py: 
import Rocket_RL.python.func.data_processing as RocketData
import Rocket_RL.python.func.ray_funcs as RF